.github/workflows/main.yml

```
@@ -0,0 +1,28 @@
+
+on:
+  pull_request_review:
+    types: [submitted, edited, dismissed]
+
+jobs:
+  forward-review:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Send PR Review to External App
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)